libs/documentation/src/lib/storybook/utilities/icons/icons.stories.ts

```diff
@@ -0,0 +1,157 @@
+        FormlyModule.forRoot(),
+        IconsColoringModule,
+        IconsRotationModule,
+        IconsScalingModule,
+        IconsSkewingModule,
+        IconsStackingModule,
+        IconsBasicModule,
+        IconsBootstrapModule,
+        IconsUswdsModule,
+      ],
+    }),
+  ],
+} as Meta;
+
+export const Coloring: Story = (args) => ({
+  template: '<usa-icon-coloring-demo></usa-icon-coloring-demo>',
+  props: args,
+});
+Coloring.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-coloring', 'IconsColoringModule', 'usa-icon-coloring-demo'),
+  stackblitzLink: generateStackblitzLink('icons-coloring', 'coloring'),
+};
+
+export const Rotating: Story = (args) => ({
+  template: '<usa-icon-rotation-demo></usa-icon-rotation-demo>',
+  props: args,
+});
+Rotating.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-rotation', 'IconsRotationModule', 'usa-icon-rotation-demo'),
+  stackblitzLink: generateStackblitzLink('icons-rotation', 'rotating'),
+};
+
+export const Sizing: Story = (args) => ({
+  template: '<usa-icon-scaling-demo></usa-icon-scaling-demo>',
+  props: args,
+});
+Sizing.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-scaling', 'IconsScalingModule', 'usa-icon-scaling-demo'),
+  stackblitzLink: generateStackblitzLink('icons-scaling', 'Sizing'),
+};
+
+export const Skewing: Story = (args) => ({
+  template: '<usa-icon-skewing-demo></usa-icon-skewing-demo>',
+  props: args,
+});
+Skewing.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-skewing', 'IconsSkewingModule', 'usa-icon-skewing-demo'),
+  stackblitzLink: generateStackblitzLink('icons-skewing', 'skewing'),
+};
+
+export const Stacking: Story = (args) => ({
+  template: '<usa-icon-stacking-demo></usa-icon-stacking-demo>',
+  props: args,
+});
+Stacking.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-stacking', 'IconsStackingModule', 'usa-icon-stacking-demo'),
+  stackblitzLink: generateStackblitzLink('icons-stacking', 'stacking'),
+};
+
+export const Custom: Story = (args) => ({
+  template: '<sds-icons-basic-demo></sds-icons-basic-demo>',
+  props: args,
+});
+Custom.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-basic', 'IconsBasicModule', 'sds-icons-basic-demo'),
```

Comment:
I would say don't provide previews or a stackblitz for the demos where every icon is being shown. There isn't a use case where they will all be shown, and we don't want a dev to accidentally copy+paste the code into production.